api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -618,4 +620,18 @@
      */
     <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 
+
+    /**
+     *
+     * Obtains an {@link Instance} giving access to beans instances
+     *
+     * Instances of dependent scoped beans obtained with this Instance should be explicitly release with {@link Instance#destroy(Object)}
+     *
+     * @return an {@link Instance} object to request beans instances
+     * @throws IllegalStateException if called during application initialization, before the {@link AfterDeploymentValidation}
+     *         event is fired.
+     * @since 2.0
+     */
+    Instance<Object> getInstance();
```

Comment:
Apart from that, originally proposed method name used plural form, 
e.g.`getInstance` -> `getInstances`